CMakeLists.txt

```diff
@@ -388,9 +394,14 @@ endforeach()
 message("")
 message("        APPLICATIONS ADDED TO COMPILATION THROUGH DEPENDENCIES:")
 message("")
-get_property(local_list GLOBAL PROPERTY LIST_OF_APPLICATIONS_ADDED_THOUGH_DEPENDENCIES)
+get_property(local_list GLOBAL PROPERTY LIST_OF_APPLICATIONS_ADDED_THROUGH_DEPENDENCIES)
 foreach(individual_app_name ${local_list})
-  message("        ${individual_app_name}")
+    if(NOT ";${LIST_OF_APPLICATIONS_TO_BE_COMPILED};" MATCHES ${individual_app_name})
+        message("        ${individual_app_name}")
+    endif(NOT ";${LIST_OF_APPLICATIONS_TO_BE_COMPILED};" MATCHES ${individual_app_name})
+    #if (${individual_app_name} IN_LIST ${LIST_OF_APPLICATIONS_TO_BE_COMPILED})
```

Comment:
I tried this but I could not make it work.

Reply:
Is not working then? looks correct :S

Reply:
Maybe its because of the separator, I remember having problems if I used spaces instead of `;`, while parsing the list maybe that changes?